lib/sisimai/bite/email/kddi.rb

```diff
@@ -0,0 +1,166 @@
+module Sisimai::Bite::Email
+  module KDDI
+    # Sisimai::Bite::Email::KDDI parses a bounce email which created by au by KDDI.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/KDDI.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->